src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -341,8 +341,12 @@ private static void EmitNativeToInterp(StreamWriter w, List<PInvokeCallback> cal
             MethodInfo method = cb.Method;
             bool isVoid = method.ReturnType.FullName == "System.Void";
 
+            if (method.DeclaringType != null && method.DeclaringType.Assembly.GetCustomAttributesData().Any(d => d.AttributeType.Name == "DisableRuntimeMarshallingAttribute"))
```

Comment:
Probably doesn't matter too much, but maybe we can keep a cache for each assembly if it has the marshaling attribute?